ZCRMiOS/Classes/ZCRMRecord.swift

```diff
@@ -754,24 +758,24 @@ public class ZCRMRecord : ZCRMEntity
         return cloneRecord
     }
     
-    public func addTags( tags : [ZCRMTag], completion : @escaping( ZCRMTag?, APIResponse?, Error? ) -> () )
+    public func addTags( tags : [ZCRMTag], completion : @escaping( [ZCRMTag]?, APIResponse?, Error? ) -> () )
```

Comment:
I think for this also we can give completion handler arguments as (APIResponse, Error) and remove [ZCRMTag] since these tag objects are not newly created objects.

Reply:
Discussed